directory/datadog_checks/directory/directory.py

```diff
@@ -48,8 +50,13 @@ def check(self, instance):
         filetagname = instance.get("filetagname", "filename")
         filegauges = _is_affirmative(instance.get("filegauges", False))
         countonly = _is_affirmative(instance.get("countonly", False))
+        ignore_missing = _is_affirmative(instance.get("ignore_missing", False))
 
         if not exists(abs_directory):
+            if ignore_missing:
+                self.log.debug("DirectoryCheck: the directory (%s) does not exist.  Skipping." % abs_directory)
```

Comment:
This should be an info log, a check shouldn't fail silently.

Reply:
Amended in.